src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -30,13 +31,13 @@ A headless browser is a great tool for automated testing and server environments
 don't need a visible UI shell. For example, you may want to run some tests against
 a real web page, create a PDF of it, or just inspect how the browser renders an URL.
 
-Note: Headless mode is available on Mac and Linux in Chrome 59. Windows support
-is coming soon!
+Note: Headless mode is available on Mac and Linux in **Chrome 59**.
+[Windows support](https://bugs.chromium.org/p/chromium/issues/detail?id=686608) is coming soon!
 
 ## Starting Headless (CLI) {: #cli }
 
 The easiest way to get started with headless mode is to open the Chrome binary
-from the command line. If you've got Chrome 59+ installed, start Chrome with the `--headless` flag:
+from the command line. If you've got **Chrome 59+** installed, start Chrome with the `--headless` flag:
```

Comment:
I think it would be too many warnings close together. These live right next to each other in the page.